core/domain/collection_domain.py

```diff
@@ -59,6 +59,10 @@
 CMD_ADD_COLLECTION_SKILL = 'add_collection_skill'
 # This takes an additional 'skill_id' parameter.
 CMD_DELETE_COLLECTION_SKILL = 'delete_collection_skill'
+# This takes an additional 'question_id', 'skill' parameter.
+CMD_ADD_QUESTION_ID_TO_SKILL = 'add_question_id_to_skill'
+# This takes an additional 'question_id', 'skill_id' parameter.
```

Comment:
This takes additional 'question_id' and 'skill' parameters.

Reply:
Updated.